src/main/java/com/rakovets/course/java/core/practice/oop/principles/hero/Enemy.java

```diff
@@ -0,0 +1,32 @@
+package com.rakovets.course.java.core.practice.oop.principles.hero;
+
+public class Enemy  implements Mortal{
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"